libs/design/docs/theming.md

```diff
@@ -0,0 +1,27 @@
+### Step 1: Add the path to your angular configuration file.
```

Comment:
All docs should start with an `#`, so in this case: `# Themeing`. Additionally, I think a brief line or two explaining the purpose of theming would be useful.